web/src/app/schedules/temp-sched/shiftsListUtil.tsx

```diff
@@ -120,26 +124,44 @@ export function getCoverageGapItems(
   const shiftIntervals = shifts.map((s) => parseInterval(s, zone))
   const gapIntervals = _.flatMap(
     schedInterval.difference(...shiftIntervals),
-    (inv) => splitAtMidnight(inv),
+    (inv) => luxonHelpers.splitAtMidnight(inv),
   )
+  const isLocalZone = zone === DateTime.local().zoneName
   return gapIntervals.map((gap) => {
     let details = 'No coverage'
+    let title = 'No coverage'
     if (gap.length('hours') === 24) {
       // nothing to do
+      title = ''
     } else if (gap.start.equals(gap.start.startOf('day'))) {
       details += ` until ${fmtTime(gap.end)}`
+      if (!isLocalZone) {
+        title += ` until ${fmtLocal(gap.end.toString())}`
+      }
     } else if (gap.end.equals(gap.start.plus({ day: 1 }).startOf('day'))) {
       details += ` after ${fmtTime(gap.start)}`
+      if (!isLocalZone) {
+        title += ` after ${fmtLocal(gap.start.toString())}`
+      }
     } else {
       details += ` from ${fmtTime(gap.start)} to ${fmtTime(gap.end)}`
+      if (!isLocalZone) {
+        title += ` from ${fmtLocal(gap.start.toString())} to ${fmtLocal(
+          gap.end.toString(),
+        )}`
+      }
     }
 
     return {
       'data-cy': 'day-no-coverage',
       id: 'day-no-coverage_' + gap.start.toISO(),
       type: 'WARNING',
       message: '',
-      details,
+      details: (
+        <Tooltip title={title} placement='right'>
```

Comment:
I think we would still need to check `isLocalZone` for the separate cases, since we want to construct respective local time strings, and they will be different for different cases.

Reply:
I think it would need to be 
`<Tooltip title={!isLocalZone ? title : ''} placement='right'>`
but yeah. I think David is saying that we can construct the title regardless of if it's the same as the local timezone or not and then just do one check at the end to see if we should use that title string or just disregard it.

Reply:
Ahh I see. Yes, constructing them regardless of timezone is local or not makes sense. Should be resolved now.